corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -407,6 +437,40 @@ def delete_all_cases(domain=None):
             cursor.execute('SELECT delete_all_cases(%s)', [domain])
 
     @staticmethod
+    def get_all_cases_modified_since(server_modified_on_since=None, chunk_size=500):
```

Comment:
yeah, definitely possible, but there seemed to be just enough custom context you'd have to pass through that it wasn't obviously worth it to me.